src/app/SimpleSubscriptionResumptionStorage.cpp

```diff
@@ -263,39 +319,39 @@ CHIP_ERROR SimpleSubscriptionResumptionStorage::SaveSubscriptions(const ScopedNo
     TLV::TLVType arrayType;
     ReturnErrorOnFailure(writer.StartContainer(TLV::AnonymousTag(), TLV::kTLVType_Array, arrayType));
 
-    for (size_t i = 0; i < subscriptions.mSize; i++)
+    for (size_t currentSubscriptionIndex = 0; currentSubscriptionIndex < subscriptions.mSize; currentSubscriptionIndex++)
```

Comment:
nit: this would read better like "subscriptionIdx". The fact it's current is pretty clear.